clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -257,6 +257,8 @@ public class VenicePushJob implements AutoCloseable {
    */
   public static final String VENICE_DISCOVER_URL_PROP = "venice.discover.urls";
 
+  public static final String VENICE_CHILD_CONTROLLER_PROP = "venice.child.controller.urls";
```

Comment:
Just want to double-check. If we have enabled the heartbeart in some jobs,  it'd require we update their config first before merging this otherwise it will fail. Have we confirmed there's no such jobs?

Reply:
Yeah, we will update the config first and then enable the heartbeat stores. Now it is disabled.